distribution/changelog.txt

```diff
@@ -1,6 +1,6 @@
 0.4.3 (in development)
 ------------------------------------------------------------------------
-- Feature: [##17782] The Flying Coaster now has access to boosters and can draw outside loops.
+- Feature: [#17782] The Flying Coaster now has access to boosters and can draw outside loops.
```

Comment:
\# is not a hashtag, it’s a hash. A hashtag is a hash followed by a tag, \#likethis.

</zeikmodus>

Reply:
Originally called an _octothorpe_. Also 'pound sign' and 'number sign'. Even calling it a "hash" is still an invention of Twitter's marketing department.

Reply:
In dutch we call it a hekje aka "little fence"